pytests/test_common_v1alpha7.py

```diff
@@ -136,71 +124,13 @@ def test_module_import_microgrid_electrical_components() -> None:
 
     assert electrical_components_pb2_grpc is not None
 
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        ev_charger_pb2,
-    )
-
-    assert ev_charger_pb2 is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        ev_charger_pb2_grpc,
-    )
-
-    assert ev_charger_pb2_grpc is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import fuse_pb2
-
-    assert fuse_pb2 is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        fuse_pb2_grpc,
-    )
-
-    assert fuse_pb2_grpc is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import grid_pb2
-
-    assert grid_pb2 is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        grid_pb2_grpc,
-    )
-
-    assert grid_pb2_grpc is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        inverter_pb2,
-    )
-
-    assert inverter_pb2 is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        inverter_pb2_grpc,
-    )
-
-    assert inverter_pb2_grpc is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        transformer_pb2,
-    )
-
-    assert transformer_pb2 is not None
-
-    # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1alpha7.microgrid.electrical_components import (
-        transformer_pb2_grpc,
-    )
-
-    assert transformer_pb2_grpc is not None
+    # pylint: disable=no-member
+    assert electrical_components_pb2.Battery is not None
+    assert electrical_components_pb2.EvCharger is not None
+    assert electrical_components_pb2.Fuse is not None
+    assert electrical_components_pb2.GridConnectionPoint is not None
+    assert electrical_components_pb2.Inverter is not None
+    assert electrical_components_pb2.VoltageTransformer is not None
```

Comment:
It looks like these tests are only for checking the generated code is reachable, we don't need to check individual structs are there.